### ISSUE TITLE
Feature Idea: Coverage Graph Based on Cards of Another Deck

### ISSUE THREAD

**Issue by ireadvns** [kampffrosch94/jpdb-deck-manager#5]
Currently I can see how the coverage of a deck changes based on the number of new cards I learn, assuming I learn them by frequency order. I often ask myself how the coverage changes based on learning the cards of another deck (again assuming freq order).

So, for example, if I learn X new cards from deck A, how would that change the coverage of deck B? If I learn all the cards of a deck which decks would increase their coverage as well? Such a feature would help a user order their decks in an optimal manner so that once they are done with a deck the next deck has the highest possible known card coverage.

**Comment by kampffrosch94** [comment]
Sorry for the late reply, I was on vacation :sweat_smile: 

Have you seen the NarrowReader tool on the site? 
It does something pretty similar to what you describe.

**Comment by KarolS** [comment]
I'm pretty sure what pinntokuru wants is the reverse of the NarrowReader.

NarrowReader says: If I want to learn deck X, how much will the other decks help learning deck X if I do them before deck X?

So NarrowReader can be used to find a way to smoothly prepare for X. The selected deck X is the finish line of learning.

A reverse NarrowReader would say: After I learn deck X, how much will it help learning other decks?

In other words, a reverse NarrowReader would predict how the decks will look like after you're finished with reading X. The selected deck X is the thing that's currently being learnt.

**Comment by kampffrosch94** [comment]
> After I learn deck X, how much will it help learning other decks?

I don't see how that is useful beyond what the narrow reader already provides.
